apps/hash-graph/tests/circular-links.http

```diff
@@ -0,0 +1,308 @@
+# This file either runs with JetBrains' http requests or using httpYac (https://httpyac.github.io).
+
+### Create account
+POST http://127.0.0.1:4000/accounts
+
+> {%
+  client.test("status", function() {
+    client.assert(response.status === 200, "Response status is not 200");
+  });
+  client.global.set("account_id", response.body.toString());
+%}
+
+### Insert entity type
+POST http://127.0.0.1:4000/entity-types
+Content-Type: application/json
+Accept: application/json
+
+{
+  "ownedById": "{{account_id}}",
+  "actorId": "{{account_id}}",
+  "schema": {
+    "$schema": "https://blockprotocol.org/types/modules/graph/0.3/schema/entity-type",
+    "kind": "entityType",
+    "$id": "http://localhost:3000/@snapshot/types/entity-type/object/v/1",
+    "type": "object",
+    "title": "Object",
+    "properties": {},
+    "links": {
+      "https://blockprotocol.org/@blockprotocol/types/entity-type/link/v/1": {
+        "type": "array",
+        "items": {
+          "oneOf": [
+            {
+              "$ref": "http://localhost:3000/@snapshot/types/entity-type/object/v/1"
+            }
+          ]
+        },
+        "ordered": false
+      }
+    }
+  }
+}
+
+> {%
+  client.test("status", function() {
+    client.assert(response.status === 200, "Response status is not 200");
+  });
+%}
+
+
+### Insert entity A
+POST http://127.0.0.1:4000/entities
+Content-Type: application/json
+Accept: application/json
+
+{
+  "ownedById": "{{account_id}}",
+  "actorId": "{{account_id}}",
+  "properties": {},
+  "entityTypeId": "http://localhost:3000/@snapshot/types/entity-type/object/v/1"
+}
```

Comment:
Just for my understanding, the `ciruclar.jsonl` snapshot has been manually cleaned up (the identifiers changed too?) Could we use `entity_uuid` as a param to alleviate some of the manual labor required after dumping the snapshot? 
Of course that means that we can't run the file multiple times, but that was the case already for `friendship.http`

Reply:
I added the `entityUuid` parameter, good flag!